scripts/sql/xavier-changes-16-03-2016/aatamsDB_queries.sql

```diff
@@ -0,0 +1,81 @@
+﻿set search_path = aatams, public;
+
+--------------------
+---- Inadvertent entry into the production db, needs to be deleted.
+DELETE FROM sensor where transmitter_id = 'A69-1303-62731' AND transmitter_type_id = 34;
+
+
+
+
+--------------------
+---- VR2-AR device model inconsistency. In the device_model table the VR2AR receiver model is stored both as 'VR2-AR' and VR2AR', it should only be stored as 'VR2AR'.
+UPDATE device SET model_id = 129558776 WHERE model_id = 117465295 -- Transforms all receivers registered as VR2-AR into VR2AR
+DELETE FROM device_model WHERE model_name = 'VR2-AR'; -- Delete wrongly named receiver model
+
+---- Test results
+-- SELECT DISTINCT receiver_name FROM detection WHERE receiver_name LIKE 'VR2-AR%' -- Returns 0 row
+-- SELECT DISTINCT receiver_name FROM detection WHERE receiver_name LIKE 'VR2AR%' -- Returns 10 rows
+-- SELECT COUNT(*) FROM device WHERE model_id = 117465295 -- VR2-AR -- Count = 20
+-- SELECT COUNT(*) FROM device WHERE model_id = 129558776 -- VR2AR -- Count = 0
+
+
+
+
+--------------------
+---- Deletion of receiver for JCU student 
+DELETE FROM receiver_deployment 
+USING device
+WHERE device.id = receiver_deployment.receiver_id 
+AND device.serial_number = '102044' AND initialisationdatetime_timestamp IS NULL;
+
+
+
+
+--------------------
+---- Add 'RANGE TEST' tag as a new type of transmitter in the transmitter_type table. TO DO: update pinger tags which we know are range test tags in the sensor table (Andre to provide list).
+INSERT INTO transmitter_type (id, version, transmitter_type_name) VALUES
+(37, 0, 'RANGE TEST');
```

Comment:
I asked this very question to @jkburges when working on this query:
"Just for our records, would you mind advising where the content of the device_model comes from? I saw that in reference_data.sql two receiver models are inserted into that table using insert queries, but can’t find how all the others got in there."

His answer was:
> They were probably inserted with ad-hoc queries, such as the one that you've suggested.
Ideally, there'd be a page in the app for admins to insert device models themselves, but it's one of the things for which there was never time!